packages/react-reconciler/src/ReactFiber.js

```diff
@@ -460,10 +460,13 @@ export function createHostRootFiber(
     }
     if (
       // We only use this flag for our repo tests to check both behaviors.
-      // TODO: Flip this flag and rename it something like "forceConcurrentByDefaultForTesting"
-      !enableSyncDefaultUpdates ||
+      !forceConcurrentByDefaultForTesting
```

Comment:
based on the name I think it needs to flip the boolean
```suggestion
      forceConcurrentByDefaultForTesting
```

Reply:
Great catch, I'm still a clown i guess